### DISCUSSION TITLE
Compiling v12.5

### DISCUSSION THREAD

**Discussion by ccfman2004** [sanni/cartreader#784] [Need help]
I see the usual 

//******************************************
// !!! CHOOSE HARDWARE VERSION !!!
//******************************************
// Remove // in front of the line with your hardware version
// #define HW5
// #define HW4
// #define HW3
// #define HW2
// #define HW1
// #define SERIAL_MONITOR

is now gone.  How do I set the HW version now?

**Comment by nsx0r** [comment]
> I see the usual
> 
> //****************************************** // !!! CHOOSE HARDWARE VERSION !!! //****************************************** // Remove // in front of the line with your hardware version // #define HW5 // #define HW4 // #define HW3 // #define HW2 // #define HW1 // #define SERIAL_MONITOR
> 
> is now gone. How do I set the HW version now?

it's in [Config.h](https://github.com/sanni/cartreader/blob/master/Cart_Reader/Config.h#L26) now

**Comment by ccfman2004** [comment]
Thank you so much.

Other question is how to remove unincluded systems from the LCD Menu?  It seems just disabling them in Config.h doesn't prevent them from appearing in the LCD menus.

**Reply by Ancyker** [reply]
You can't at present. I am working on a way to do this, however, it's not ready yet. Got a lot on my plate.

**Reply by ccfman2004** [reply]
No worries. Just annoying there's a bunch of systems my setup can't dump in the menu.

**Reply by Chomemel** [reply]
I never thought about that. But that could be very neat.
